src/Init/Control/Lawful/MonadLift.lean

```diff
@@ -0,0 +1,11 @@
+/-
+Copyright (c) 2021 Microsoft Corporation. All rights reserved.
```

Comment:
How did you come up with these copyright headers? The file you're upstreaming is `Copyright (c) 2025 Quang Dao.`.

Reply:
I think they are from `Init.Control.Lawful.Basic`, where the code lived for some time. Will fix.

Reply:
I changed the copyright headers so that they are like they were in Batteries, I hope that's the right way.